orchagent/port.h

```diff
@@ -11,7 +11,7 @@ extern "C" {
 #include <map>
 
 #define DEFAULT_PORT_VLAN_ID    1
-#define DEFAULT_MTU              9100
+#define DEFAULT_MTU             1492
```

Comment:
>define [](start = 1, length = 6)

If you are fixing a bug, could you add a VS test?

Reply:
agree

Reply:
Had a discussion with Guohan, VS test case would be added in the next commit since it might take some time. This change is mainly to unblock the MTU test failures.